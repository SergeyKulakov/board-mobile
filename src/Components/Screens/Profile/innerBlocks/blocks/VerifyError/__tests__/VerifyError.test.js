import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { VerifyError as Component } from '../index'

describe('Profile.VerifyError', () => {
  const defaultProps = {
    text: {
      info: 'info',
      link: 'link',
    },
    loading: false,
    onClick: () => null,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
