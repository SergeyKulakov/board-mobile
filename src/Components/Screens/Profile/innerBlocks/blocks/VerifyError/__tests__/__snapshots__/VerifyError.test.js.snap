// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile.VerifyError render snapshot 1`] = `
<Styled(View)>
  <Styled(Text)>
    info
  </Styled(Text)>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Styled(Styled(Text))>
      link
    </Styled(Styled(Text))>
  </TouchableOpacity>
</Styled(View)>
`;
